index.html

```diff
@@ -691,6 +691,26 @@ <h2>Verifiable Credential</h2>
         </aside>
       </section>
     </section>
+
+    <section>
+      <h2>Securing JSON-LD</h2>
+      <section>
+        <h2>Securing <code>application/credential+ld+json</code> with JOSE</h2>
+        <p>[[rfc7515]] MAY be used to secure this media type.</p>
+        <p><code>typ</code> MUST be <code>vc+ld+jwt</code></p>
+        <p><code>cty</code> MUST be <code>application/credential+ld+json</code></p>
+        <p>See <a data-cite="rfc7515#section-4.1.10">Common JOSE Header Parameters</a> for additional details.</p>
+      </section>
+      <section>
+        <h2>Securing <code>application/credential+ld+json</code> with COSE</h2>
+        <p>[[rfc8152]] MAY be used to secure this media type.</p>
+        <p><code>type (TBD)</code> MUST be <code>vc+ld+cwt</code></p>
+        <p><code>content type (3)</code> MUST be <code>application/credential+ld+json</code></p>
+        <p>See <a data-cite="rfc9052#section-3.1">Common COSE Header Parameters</a> for additional details.</p>
+        <p>See <a href="https://www.iana.org/assignments/cbor-tags/cbor-tags.xhtml">Concise Binary Object Representation (CBOR) Tags</a> for additional details.</p>
+      </section>
```

Comment:
```suggestion
      </section>
      <section>
        <h2>Presence of JSON-LD in a credential</h2>
        <p>If a JSON-LD context is present and indicated by use of one of the above `vc+ld` media types, the JSON-LD MUST be valid JSON-LD.</p>
        <p>This requirement is highly simplified by use of `@vocab` in the base context for verifiable credentials, which permit the use of private claims as a part of valid JSON-LD.  This further ensures that semantics are attached to the core data model elements at a minimum, even if the implementer does not utilize additional or inline vocabularies.</p>
      </section>
```

Reply:
I'd prefer direct citation to relevant specs and less rehashing them here.

Reply:
yeah - i think we should pull 
```
<p>This requirement is highly simplified by use of `@vocab` in the base context for verifiable credentials, which permit the use of private claims as a part of valid JSON-LD.  This further ensures that semantics are attached to the core data model elements at a minimum, even if the implementer does not utilize additional or inline vocabularies.</p>
```

doing so, and adjusting the first paragraph in the suggestion that i think is important to prevent things that are 'implied' by other specs from being accidentally overlooked

Reply:
on reflection this likely belongs in the core data model spec - added to [data model pr #1033](https://github.com/w3c/vc-data-model/pull/1033)